docs: Update Moderator Handbook (#40110)
freeCodeCamp/freeCodeCamp: docs/moderator-handbook.md

1. **Challenge Instruction Edits** These are changes to the text of challenges - the Description, Instructions, or Test Text. You can also review these right on GitHub and decide whether to merge them. We need to be a bit more careful about these, because millions of people will encounter this text as they work through the freeCodeCamp curriculum. Does the pull request make the text more clear without making it much longer? Are the edits relevant and not overly pedantic? Remember that our goal is for challenges to be as clear and as short as possible. They aren't the place for obscure details. Also, contributors may try to add links to resources to the challenges. You can close these pull requests and reply to them with this:

```markdown
Thank you for your pull request.
 

> I am closing this pull request. Please add links and other details to the challenge's corresponding guide article instead.

Reviewer: ```suggestion
We are closing this pull request. Please add links and other details to the challenge's corresponding guide article instead.
```

> If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you, and happy coding.

Reviewer: ```suggestion
If you think we're wrong in closing this issue, please request for it to be reopened and add further clarification. Thank you, and happy coding.
```

3. **Duplicate Issues** If an issue is the same as another reported issue, the prior reported issue should take precedence. Flag as `Duplicate`, paste the following message replacing `#XXXXX` with the issue number, then close the issue.

```markdown
Thank you for reporting this issue.

> This is a standard message notifying you that this issue appears to be very similar to issue #XXXXX, so I am closing it as a duplicate.

Reviewer: ```suggestion
This is a standard message notifying you that this issue appears to be very similar to issue #XXXXX, so we are closing it as a duplicate.
```

> If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you and happy coding.

Reviewer: ```suggestion
If you think we're wrong in closing this issue, please request for it to be reopened and add further clarification. Thank you and happy coding.
```

```

4. **Fixed in staging** Some problems may have already been fixed in staging, but don't have a GitHub issue associated with them. If this is the case, you can paste the following message, close the issue, and add a `status: resolved/shipping` label:

```markdown
Thank you for reporting this issue.

> This is a standard message notifying you that the problem you mentioned here is present in production, but that it has already been fixed in staging. This means that the next time we push our staging branch to production, this problem should be fixed. Because of this, I'm closing this issue.

Reviewer: ```suggestion
This is a standard message notifying you that the problem you mentioned here is present in production, but that it has already been fixed in staging. This means that the next time we push our staging branch to production, this problem should be fixed. Because of this, we're closing this issue.
```

> If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you and happy coding.

Reviewer: ```suggestion
If you think we're wrong in closing this issue, please request for it to be reopened and add further clarification. Thank you and happy coding.
```

Thank you and happy coding.
```

> When PR adds links to external resources.

```markdown
Thank you for your pull request.

> I am closing this pull request. Please add links and other details to the challenge's corresponding guide article instead.

Reviewer: ```suggestion
We are closing this pull request. Please add links and other details to the challenge's corresponding guide article instead.
```

> If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you, and happy coding.

Reviewer: ```suggestion
If you think we're wrong in closing this issue, please request for it to be reopened and add further clarification. Thank you, and happy coding.
```

```

## Closing Invalid Issues

> When an issue relates to the camper's code.

```markdown
Thank you for reporting this issue.

This is a standard message notifying you that this issue seems to be a request for help. Instead of asking for help here, please click the **"Help"** button on the challenge on freeCodeCamp, which will help you create a question in the right part of the forum. Volunteers on the forum usually respond to questions within a few hours and can help determine if there is an issue with your code or the challenge's tests.

If the forum members determine there is nothing wrong with your code, you can request this issue to be reopened.

Thank you and happy coding.
```

> When an issue is duplicate of an earlier issue

```markdown
Thank you for reporting this issue.

> This is a standard message notifying you that this issue appears to be very similar to issue #XXXXX, so I am closing it as a duplicate.

Reviewer: ```suggestion
This is a standard message notifying you that this issue appears to be very similar to issue #XXXXX, so we are closing it as a duplicate.
```

> If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you and happy coding.

Reviewer: ```suggestion
If you think we're wrong in closing this issue, please request for it to be reopened and add further clarification. Thank you and happy coding.
```

```

> When an issue is fixed in staging.

```markdown
Thank you for reporting this issue.

> This is a standard message notifying you that the problem you mentioned here is present in production, but that it has already been fixed in staging. This means that the next time we push our staging branch to production, this problem should be fixed. Because of this, I'm closing this issue.

Reviewer: ```suggestion
This is a standard message notifying you that the problem you mentioned here is present in production, but that it has already been fixed in staging. This means that the next time we push our staging branch to production, this problem should be fixed. Because of this, we're closing this issue.
```

> If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you and happy coding.

Reviewer: ```suggestion
If you think we're wrong in closing this issue, please request for it to be reopened and add further clarification. Thank you and happy coding.
```
